app/templates/discord.hbs

```diff
@@ -19,32 +19,48 @@
           <p class="error__text">Something went wrong. Please try again.</p>
         </div>
       {{else if (eq this.linkStatus "not-linked")}}
+        <div class="linking__logo">
+          <img src="RDSLogo.png" alt="real_dev_squad" height="100px" width="100px" />
+          <div class="line"></div>
+          <img src="DiscordLogo.png" alt="real_dev_squad" height="100px" width="100px" />
+        </div>
         <h1 class="discord__title">Authorize Bot</h1>
+        <div class="container">
```

Comment:
container here is not telling anything meaningful like what is it is used for. May be we should try changing the name to something more relatable